### ISSUE TITLE
Create a Kubernetes skill tree 

### ISSUE THREAD

**Issue by remyleone** [kubernetes/website#18887]
**This is a Feature Request**

**What would you like to be added**

I would like to add a skill tree in Kubernetes. 
Like in video games where you have skills that are unblocked after you acquire other skills. The graph of interdependencies could be a guided way to explore Kubernetes in a way where new users know where they are and what they should learn next to solve a set of problems.
The content of the different skills would be in English and then localized in different languages.
We can have a different path depending on whether you are in a developer position, in an ops position and so on...
Because Kubernetes move a lot, it has to be generated to be able to be edited and rendered quickly.

![image](https://user-images.githubusercontent.com/300834/73210737-2823aa00-414b-11ea-9da1-e64c678b2975.png)

**Why is this needed**

It is relatively easy for a beginner to get lost quickly on the learning of Kubernetes.
You will find plenty of tutorials about how to get started but it doesn’t constitute a roadmap of stuff to learn and the dependencies between the different concepts and tasks.
Having a roadmap could be very helpful to suggest a new concept to explore.
Books and courses already have it on their own but I think it would be nice to have an official learning roadmap that could be suggested to learners and improved by the whole community.

**Comments**

Ideally, I would like to have something like this https://github.com/kamranahmedse/developer-roadmap
but applied to Kubernetes concepts/tasks/etc...
